tagstudio/src/qt/ts_qt.py

```diff
@@ -1030,25 +1030,41 @@ def update_thumbs(self):
         self.flow_container.layout().update()
         self.main_window.update()
 
-        for idx, (entry, item_thumb) in enumerate(
-            zip_longest(self.frame_content, self.item_thumbs)
-        ):
+        is_grid_thumb = True
+        # Show loading placeholder icons
+        for entry, item_thumb in zip_longest(self.frame_content, self.item_thumbs):
             if not entry:
                 item_thumb.hide()
                 continue
 
-            filepath = self.lib.library_dir / entry.path
-            item_thumb = self.item_thumbs[idx]
             item_thumb.set_mode(ItemType.ENTRY)
             item_thumb.set_item_id(entry)
 
             # TODO - show after item is rendered
             item_thumb.show()
 
+            is_loading = True
```

Comment:
could this not be hardcoded? As far as I can tell this is never changed and only accessed once
(same in the next loop)

Reply:
It *can* be hard-coded, but the reason I split it off into a variable is due to the way the arguments are being passed where I'm unable to use named parameters. It took some time to sort out which of the existing unnamed boolean arguments were doing what here, so I opted to store them in named variables for clarity.